test/core.sh

```diff
@@ -119,3 +120,7 @@ echo "${_endgroup}"
 echo "${_group}Ensure cleanup crons are working ..."
 $dc ps | grep -q -- "-cleanup_.\+[[:space:]]\+Up[[:space:]]\+"
 echo "${_endgroup}"
+
+echo "${_group}Test customs CAs work ..."
```

Comment:
```suggestion
echo "${_group}Test custom CAs work ..."
```

Reply:
Too much poking and prodding government paper work. >.> Fixed.